modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -52,3 +57,57 @@ To resolve this issue:
 . Delete the API.
 . Resolve the username and password error.
 . Re-create the API.  
+
+
+[[provision-call-failed]]
+== API Provisioning Failure
+
+When you provision an API by using Anypoint Service Mesh, the asset (in Exchange) and the API instance (in API Manager) might not be successfully created in Anypoint Platform.
+
+=== Causes
+
+This error can occur if you are attempting to:
+ 
+* Re-create an asset in Exchange that was more than seven days old before being deleted
+* Provision an API with invalid values
+
+
+=== Diagnose
+
+To diagnose this issue: 
+
+. Run the `$ asmctl api list` command. 
++
+The status of the API is displayed as `ProvisionCallFailed`. 
+. Run the `asmctl api logs --name=<replace_with_api_name> --namespace=<replace_with_namespace>` command.
++
+The following output illustrates provisioning an API with an invalid version:
+
+[source,text,linenums]
+----
+DATE                   MESSAGE
+2020-05-29T17:12:36Z   Error provisioning ServiceInstance of ClusterServiceClass (K8S: "1c3b2100-5cdc-49ea-9423-dd7170e674b4" ExternalName: "anypoint-platform-api-instance") at ClusterServiceBroker "service-mesh-incluster-broker": Status: 400; ErrorMessage: Bad Request; Description: version: should match pattern "^\d+\.\d+\.\d+(-.+)?$" (ID: 49724b5d-7fad-4cc3-980e-695c2e47b383); ResponseError: <nil>
+----
+
+If no information is displayed, run the `kubectl describe serviceinstances <replace_with_api_name> -n <replace_with_namespace>` command. 
```

Comment:
I'm still nervous that this text constitutes a "hidden" step (user action), but I accept your decision!

Reply:
Converted it to a step. Sorry, I misunderstood that one. Done! Thanks!